src/test/java/org/reactome/addlinks/test/TestTargetPathogenFileProcessor.java

```diff
@@ -0,0 +1,54 @@
+package org.reactome.addlinks.test;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Map;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.reactome.addlinks.fileprocessors.TargetPathogenFileProcessor;
+
+public class TestTargetPathogenFileProcessor
+{
+	private Path pathToFile;
+
+	private final static String fileContent = "http://target.sbg.qb.fcen.uba.ar/patho/protein/56425908be737e6c7a9fd4d0,H37Rv,Rv0046c|ino1,R-MTU-879331,P9WKI1\n" + 
+			"http://target.sbg.qb.fcen.uba.ar/patho/protein/56425908be737e6c7a9fd525,H37Rv,Rv0126|treS,R-MTU-868709,P9WQ19\n" + 
+			"http://target.sbg.qb.fcen.uba.ar/patho/protein/5b2801b3be737e35a6df1a69,MeloI,Minc3s11049g44572,R-CEL-74723,P34462\n" + 
+			"http://target.sbg.qb.fcen.uba.ar/patho/protein/5b2801b3be737e35a6df151a,MeloI,Minc3s09360g43141|RP-L40e,|RPL40,R-XTR-983157,F6VNK4\n" + 
+			"http://target.sbg.qb.fcen.uba.ar/patho/protein/5b2801b3be737e35a6df151a,MeloI,Minc3s09360g43141|RP-L40e,|RPL40,R-CFA-983157,F1PDG4\n" + 
+			"http://target.sbg.qb.fcen.uba.ar/patho/protein/5b354852be737e166372e8dd,EcoAIEC_C7225,AIEC_C7225_05581|APU18_05390| pNDM102337_153|groEL| MS6198_A148| CA268_28940| pNDM10505_155|BET08_16250| NDM1Dok01_N0169,R-HSA-8850539,P48643\n" + 
+			"http://target.sbg.qb.fcen.uba.ar/patho/protein/5b354852be737e166372e8dd,EcoAIEC_C7225,AIEC_C7225_05581|APU18_05390| pNDM102337_153|groEL| MS6198_A148| CA268_28940| pNDM10505_155|BET08_16250| NDM1Dok01_N0169,R-SSC-8850539,I3LR32\n";
```

Comment:
Could the "http://target.sbg.qb.fcen.uba/ar/patho/protein/" part of the URL be extracted into a constant so it's not repeated?

It also may be clearer to use String.join("\n", ...) instead of concatenating all the Strings with a new-line at the end of each.

Reply:
Ok, I'll refactor the URL.

I usually avoid using `String.join` for situations that involve string literals.

Reply:
Okay, that's fine.